src/Compiler/Checking/infos.fs

```diff
@@ -866,9 +866,7 @@ type MethInfo =
     member x.IsDispatchSlot =
         match x with
         | ILMeth(_g, ilmeth, _) -> ilmeth.IsVirtual
-        | FSMeth(g, _, vref, _) as x ->
-            isInterfaceTy g x.ApparentEnclosingType  ||
-            vref.MemberInfo.Value.MemberFlags.IsDispatchSlot
+        | FSMeth(_, _, vref, _) -> vref.MemberInfo.Value.MemberFlags.IsDispatchSlot
```

Comment:
This would trigger the static abstract warning when calling a concrete static method, even though the interface had no static abstract methods.

Reply:
Could you please add test for it if you haven't already?

Reply:
![image](https://user-images.githubusercontent.com/5063478/216678165-3e03a6b8-27e6-428e-84c7-35cd0cf9ebeb.png)

this still passes and my tests don't raise a warning, so it should be fine.